src/logging/Initialisation.cpp

```diff
@@ -0,0 +1,61 @@
+#include "Initialisation.h"
+#include <boost/log/core.hpp>
+#include <boost/log/trivial.hpp>
+#include <boost/log/expressions.hpp>
+#include <boost/log/utility/setup/common_attributes.hpp>
+#include <boost/log/sources/record_ostream.hpp>
+#include <boost/log/attributes/mutable_constant.hpp>
+#include <boost/log/support/date_time.hpp>
+#include <boost/log/sinks/sync_frontend.hpp>
+#include <boost/log/sinks/text_ostream_backend.hpp>
+#include <boost/core/null_deleter.hpp>
+#include <boost/filesystem.hpp>
+#include <boost/smart_ptr/shared_ptr.hpp>
+#include <boost/smart_ptr/make_shared_object.hpp>
+
+namespace log = boost::log;
+typedef log::sinks::synchronous_sink<log::sinks::text_ostream_backend> text_sink;
+
+void Formatter(log::record_view const& rec, log::formatting_ostream& stream) {
```

Comment:
for all this functions, you can create a class that will init on ctor, working RAII style